File: app/Touch_Game/app/src/main/java/com/example/touch_game/MainActivity.java
package com.example.touch_game;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.example.touch_game.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    ActivityMainBinding binding;
    int count = 10;
    int power = 1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.countTouchTxt.setText(String.valueOf(count));

        binding.touchImg.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                count -= power;
                if(count == -1){
                    count = 10;
                }
                binding.countTouchTxt.setText(String.valueOf(count));
            }
        });
    }
}

File: App/todolist/app/src/main/java/com/example/todolist/MainActivity.kt
package com.example.todolist

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.EditText
import android.widget.Toast
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.example.todolist.adapter.NoteAdapter
import com.example.todolist.data.Note
import com.example.todolist.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    val binding by lazy { ActivityMainBinding.inflate(layoutInflater)}

    val itemlist = ArrayList<Note>()
    lateinit var mainFragment: MainFragment

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)

        val pref = getSharedPreferences("COUNT", MODE_PRIVATE)
        val count = pref.getInt("count", 0)

        mainFragment = MainFragment(itemlist)
        supportFragmentManager.beginTransaction().replace(R.id.container, mainFragment).commit()
        binding.saveBtn.setOnClickListener {
            val edit = pref.edit()
            itemlist.add(Note(count, findViewById<EditText>(R.id.input_to_do).text.toString()))
            edit.putInt("count", count+1)
            mainFragment = MainFragment(itemlist)
            supportFragmentManager.beginTransaction().replace(R.id.container, mainFragment).commit()
        }
    }
}